interface/modules/custom_modules/oe-module-weno/src/Services/LogProperties.php

```diff
@@ -151,13 +163,66 @@ public function logSync()
             file_put_contents($this->rxsynclog, $rpt);
             $logstring = "prescription log import initiated successfully";
             EventAuditLogger::instance()->newEvent("prescriptions_log", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "$logstring");
+            $wenolog->insertWenoLog("prescription", "Success");
         } else {
             EventAuditLogger::instance()->newEvent("prescriptions_log", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "$statusCode");
+            $wenolog->insertWenoLog("prescription", "Failed");
+            return false;
         }
 
         if (file_exists($this->rxsynclog)) {
             $log = new LogImportBuild();
             $log->buildInsertArray();
+            return true;
+        }
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getProviderEmail()
+    {
+        if($_SESSION['authUser']){
+            $provider_info = sqlQuery("select setting_value AS email from user_settings where setting_user = ? and setting_label = 'global:weno_provider_email'", [$_SESSION["authUserID"]]);
+            if(!empty($provider_info)){
+                return $provider_info;
+            } else {
+                $error = "Provider email address is missing. Go to [User settings > Email] to add provider's weno registered email address";
+                error_log($error);
+                exit;
+            }
+        }
+        else if($GLOBALS['weno_admin_username']){
+            $provider_info["email"] = $GLOBALS['weno_admin_username'];
+            return $provider_info;
+        }
+        else{
+            $error = "Provider email address is missing. Go to [User settings > Email] to add provider's weno registered email address";
+            error_log($error);
+            exit;
+        }
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getProviderPassword()
+    {
+        if($_SESSION['authUser']){
+            $uid = $_SESSION['authUserID'];
+            $sql = "select setting_value from user_settings where setting_user = ? and setting_label = 'global:weno_provider_password'";
```

Comment:
user settings are stored in globals (they will overwrite the main global if it exists), so should be able to get above by just using $GLOBALS['weno_provider_password']